src/main/java/cz/muni/fi/pa165/bluebat/dao/CustomerDaoImpl.java

```diff
@@ -0,0 +1,56 @@
+package cz.muni.fi.pa165.bluebat.dao;/* created by rudolf */
```

Comment:
Takhle to dělá přímo IDE? Myslím, že odsud můžeš ten koment smazat, informace o autorovi už je nad třídou